assignment/api/src/resources/assignment_eu.properties

```diff
@@ -266,7 +266,7 @@ gen.hideduedate=Ezkutatu emate-data ikasleei
 grad2=Sartutako nota, "{0}", gehienezko puntuazioa baino handiagoa da, hau da, "{1}". Zuzendu gaizki badago, bestela, gordetzeko botoia sakatu emandako nota berresteko, "{0}".
 
 grad3=Kalifikatua
-
+grade.title=Balorazioak
```

Comment:
Good morning @pablosanroman , could you take a look at these basque strings?

Thanks